src/test/scala/no/ndla/imageapi/service/search/SearchServiceIntegrationTest.scala

```diff
@@ -213,6 +213,22 @@ class SearchServiceIntegrationTest extends UnitSuite with TestEnvironment {
     searchResult.results.head.license should equal(agreement1Copyright.license.license)
   }
 
+  test("Searching for multiple languages should returned matched language") {
+    val searchResult1 = searchService.matchingQuery("urelatert", None, Some("all"), None, Sort.ByIdAsc, None, None, false)
+    searchResult1.totalCount should be (1)
+    searchResult1.results.size should be (1)
+    searchResult1.results.head.id should be ("5")
+    searchResult1.results.head.title.language should equal("unknown")
+    searchResult1.results.head.altText.language should equal("unknown")
+
+    val searchResult2 = searchService.matchingQuery("unrelated", None, Some("all"), None, Sort.ByTitleDesc, None, None, false)
+    searchResult2.totalCount should be (1)
+    searchResult2.results.size should be (1)
+    searchResult2.results.head.id should be ("5")
+    searchResult2.results.head.title.language should equal("en")
+    searchResult2.results.head.altText.language should equal("en")
+  }
+
```

Comment:
Hmm. Må du alltid spørre etter "alle" språk? Hva med de tilfellene du har et språk men du mangler innslag på det språket og derfor får innslag "unknown" eller noe annet. Mao. en slik test:
```
    val searchResult3 = searchService.matchingQuery("urelatert", None, Some("no"), None, Sort.ByIdAsc, None, None, false)
    searchResult3.totalCount should be (1)
    searchResult3.results.size should be (1)
    searchResult3.results.head.id should be ("5")
    searchResult3.results.head.title.language should equal("unknown")
    searchResult3.results.head.altText.language should equal("unknown")
```

Men det er mulig jeg ikke helt skjønner hvordan dette brukes fra klientside... 
Jeg skjønner plutselig ikke helt hvorfor dette er en del av selve søket, og ikke bare en prioritering av "språk-felter" på et allerede utført søk...

Reply:
Vet ikke om det finnes en test på det, men det er funksjonalitet som er i bilde-api'et fra før av.
Tenkte å lage en annen pull-request som gjør prioriteringen slik som nevnt i https://github.com/NDLANO/Issues/issues/742

Reply:
Ah. Ok. Men fint om du legger inn den testen for å vise at funksjonaliteten er der...

Reply:
Fant ut at det er jeg som surrer. 
Dette skjer bare når du henter ut et enkelt bilde, ikke når du søker.
Da kommer det til søket sammen med prioriteringen i #742.